exporter/datadogexporter/testdata/config.yaml

```diff
@@ -8,8 +8,6 @@ exporters:
   datadog/api:
     hostname: customhostname
     env: prod
-    service: myservice
-    version: myversion
```

Comment:
We're deprecating these options, but they're technically still supported until v0.51.0. Should we keep the tests until we definitely remove these options (to prevent breaking the behavior of these options before the scheduled date)?

Reply:
I added back this with a comment stating they are deprecated. I didn't do this originally because you need to check Config fields individually until we do #8373 (since the `warnings` field is unexported)